client/view/base/base.css

```diff
@@ -30,3 +30,7 @@ code {
   line-height: 22px;
   font-weight: 600;
 }
+
+[hidden] {
+  display: none !important;
```

Comment:
Can we avoid `!important` to have clean code?

Reply:
`*[hidden]`?

Reply:
`*[hidden]* is `(0,1,0)`

The `*` has zero specificity :(

Reply:
Do we have any rational reasons for `[hidden]`?

I like it, but need some reason for `!important`.

Reply:
This saves us from duplicating simple `display: none` `*--hide` in BEM.

It seems that this is the same as the `.is-hidden` class, it's just more convenient to call it in CSS.

Reply:
We can make a CSS module "css utility", which will be connected at the very bottom of "index.css". However, this may break when refactoring.

Reply:
We can make a CSS module `view/base/css-utility.css` with `[hidden] { display: none }` , which will be connected at the bottom of `client/index.css` entry. However, this may break when refactoring.

Reply:
OK, let’s keep it with `!important` as we have `animation: none` for `reduce-motion`.

Reply:
Or we can remove this change :)

From the point of view of accessibility, hidden works the same way as `display: none`

Reply:
ok :)